File: async-scala-client/src/main/scala/io/swagger/client/model/IcmsTaxConfBase.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class IcmsTaxConfBase (
  state: StateEnum,
icmsCST: Option[String],  // On sales process inform the CST hat the item linked to this configuration is subject to - &#39;00&#39; # TAXABLE - &#39;20&#39; # TAXABLE WITH BASE DISCOUNT - &#39;40&#39; # EXEMPT - &#39;41&#39; # NOT TAXABLE - &#39;50&#39; # SUSPENDED 
messageCode: Option[String]  // Message to add to NF when this configuration is used
)

File: javascript-closure-angular-client/API/Client/IcmsTaxConfBase.js
goog.provide('API.Client.IcmsTaxConfBase');

/**
 * @record
 */
API.Client.IcmsTaxConfBase = function() {}

/**
 * @type {!API.Client.StateEnum}
 * @export
 */
API.Client.IcmsTaxConfBase.prototype.state;

/**
 * On sales process inform the CST hat the item linked to this configuration is subject to - '00' # TAXABLE - '20' # TAXABLE WITH BASE DISCOUNT - '40' # EXEMPT - '41' # NOT TAXABLE - '50' # SUSPENDED 
 * @type {!string}
 * @export
 */
API.Client.IcmsTaxConfBase.prototype.icmsCST;

/**
 * Message to add to NF when this configuration is used
 * @type {!string}
 * @export
 */
API.Client.IcmsTaxConfBase.prototype.messageCode;

/** @enum {string} */
API.Client.IcmsTaxConfBase.IcmsCSTEnum = { 
  00: '00',
  20: '20',
  40: '40',
  41: '41',
  50: '50',
}
